snuba/migrations/groups.py

```diff
@@ -15,6 +16,23 @@ class MigrationGroup(Enum):
     QUERYLOG = "querylog"
 
 
+# Migration groups are mandatory by default, unless they are on this list
+OPTIONAL_GROUPS = {
+    MigrationGroup.TRANSACTIONS,
+    MigrationGroup.OUTCOMES,
+    MigrationGroup.SESSIONS,
```

Comment:
I am not sure I would make these optional. The on prem standard install of sentry strictly depends on them.

Reply:
I'm ok with that. Thoughts on just leaving sessions on this list together with querylog? Reason being that single tenant does not run sessions so it'd be nice to skip the migrations.

Reply:
ok